pkg/metrics/metrics.jsx

```diff
@@ -867,6 +872,92 @@ class MetricsHour extends React.Component {
     }
 }
 
+const persistentServiceState = proxy => ['running', 'stopped', 'failed', undefined].indexOf(proxy.state) >= 0;
+
+const PCPConfig = ({ buttonVariant }) => {
+    const [dialogVisible, setDialogVisible] = useState(false);
+    const [dialogError, setDialogError] = useState(null);
+    const [dialogLoggerValue, setDialogLoggerValue] = useState(false);
+    const [pending, setPending] = useState(false);
+    const s_pmlogger = useObject(() => service.proxy("pmlogger.service"), null, []);
+
+    debug("PCPConfig s_pmlogger.state", s_pmlogger.state);
+
+    useEvent(s_pmlogger, "changed");
+    useEvent(superuser, "changed");
+
+    if (!superuser.allowed)
+        return null;
+
+    const handleApply = () => {
+        setPending(true);
+        if (dialogLoggerValue) {
+            Promise.all([s_pmlogger.start(), s_pmlogger.enable()])
+                    .then(() => setDialogVisible(false))
+                    .catch(err => setDialogError(err.toString()))
+                    .finally(() => setPending(false));
+        } else {
+            Promise.all([s_pmlogger.stop(), s_pmlogger.disable()])
+                    .then(() => setDialogVisible(false))
+                    .catch(err => setDialogError(err.toString()))
+                    .finally(() => setPending(false));
+        }
+    };
+
+    return (
+        <>
+            <Button variant={buttonVariant} icon={<CogIcon />}
+                    isDisabled={ !persistentServiceState(s_pmlogger) }
+                    onClick={ () => {
+                        setDialogLoggerValue(s_pmlogger.state === 'running');
+                        setDialogVisible(true);
+                    } }>
+                { _("Metrics settings") }
+            </Button>
+
+            {dialogVisible &&
+            <Modal position="top" variant="small" isOpen
+                   id="pcp-settings-modal"
+                   onClose={ () => setDialogVisible(false) }
+                   title={ _("Metrics settings") }
+                   description={
+                       <div className="pcp-settings-modal-text">
+                           { _("Performance Co-Pilot collects and analyzes performance metrics from your system.") }
+
+                           <Button component="a" variant="link" href="https://cockpit-project.org/guide/latest/feature-pcp.html"
+                                isInline
+                                target="_blank" rel="noopener noreferrer"
+                                icon={<ExternalLinkAltIcon />}>
+                               { _("Read more...") }
+                           </Button>
+                       </div>
+                   }
+                   footer={<>
+                       { dialogError && <ModalError dialogError={ _("Failed to configure PCP") } dialogErrorDetail={dialogError} /> }
+
+                       <Button variant='primary' onClick={handleApply} isDisabled={pending}>
+                           { _("Apply") }
```

Comment:
```suggestion
                           { _("Save") }
```

This should be "Save", not "Apply".